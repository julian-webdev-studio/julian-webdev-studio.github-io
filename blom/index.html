<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>Bloom-eComerce</title>
	<link rel="stylesheet" type="text/css" href="assets/css/style.css">
	<link href="https://fonts.googleapis.com/css2?family=Signika+Negative:wght@300;400;600;700&display=swap" rel="stylesheet">
	<link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital@1&display=swap" rel="stylesheet">
</head>
<body>
	<div class="main">
	<header>
		<div class="search">
			<h2>
				search
			</h2>
			<a href="#"><img src="assets/img/Loupe_Icon.png" alt="loupeicon"></a>
		</div>
		<div class="bag"><a href="#"><img src="assets/img/Bag.png" alt=""></a></div>
		<div class="user">
			<a href="#" ><img src="assets/img/3-layers.png" alt=""></a>
		</div>
		<div class="logo">
			<h1><a href="index.html">bloom</a></h1>
			<h2>co.ncept</h2>
		</div>

		<div class="menubar">		
				<ul class="menu">
					<li><a href="#">Lighting</a></li>
					<li><a href="#">Wearables</a></li>
					<li><a href="#">Bicycle</a></li>
					<li><a href="#">Accessory</a></li>
					<li><a href="#">Furniture</a></li>
				</ul>
		</div>
	</header>

	<div class="site_cont">
		
		<div class="site_cont1">
			<div class="cont1_element">
				<div class="site_cont1_txt">
					<h2>a new generation of saddles</h2>
				</div>
			</div>
			<h2 class="cambium">Cambium is crafted from high-quality materials</h2>
			<h2 class="go_sale_txt"><a href="#">Go to sale it ></a></h2>
		</div>
		<div class="site_cont2">
			<div class="site_cont_img">
			</div>
			<h2> The Pack</h2>
			<hr>
			<p>The Easy Slicer is a very clever redesign for a canned ham packaging</p>
			<div class="like_btn"><h2><a href="#">like it</a></h2></div>
		</div>
		</div>
		<div class="pop_prod">
			<h2>POPULAR PRODUCTS</h2>
			<hr class="first_hr">
			<hr class="second_hr">
		</div>
		<div class="site_cont3">
			<div class="site_cont3_elem1"></div>
			<div class="site_cont3_elem2">
				<div class="site_cont3_search">
					<a href="#"><img src="assets/img/Magnifying_Glass.png" alt="&#128269;"></a>
				</div>
				<h2 class="site_cont3_elem2_txt">cambium is crafted</h2>
				<a href="#"><h2 class="site_cont3_elem2_txt2">look closer</h2></a>
				<hr>
			</div>
			<div class="site_cont3_elem3"></div>
			<div class="site_cont3_elem4"></div>
			<div class="site_cont3_elem5"></div>
			<div class="site_cont3_elem6"></div>
		</div>
		<footer>
			<ul class="footer_nav">
				<li><a href="#">About us</a></li>
				<li><a href="#">Shipping Rates</a></li>
				<li><a href="#">Shop with Points</a></li>
				<li><a href="#">Help</a></li>
			</ul>
			<div class="footer_h2_p">
				<div class="footer-logo"><h2>bloom.co</h2></div>
				<div class="footer-paragraph"><p>There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected </p></div>
			</div>
			<div class="social-media">
				<ul class="social-media-links">
					<li><a href="#">Facebook </a></li>
					<li><a href="#">Twitter </a></li>
					<li><a href="#">Pinterest</a></li>
				</ul>
			</div>
			
		</footer>
	</div>
</body>
</html>   