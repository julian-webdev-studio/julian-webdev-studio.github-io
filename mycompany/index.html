<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>MyCompany</title>
    <link rel="stylesheet" href="assets/css/style.css">
    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;800&display=swap" rel="stylesheet">
</head>
<body>
    <header>
        <div class="header__logo">
            <div id="header__link"></div>
          <h1><a href="#">mycompany</a></h1>
        </div>
        <nav>
            <div class="topnav" id="mytopnav">
                <a href="#">home</a>
                <a href="#">about</a>
                <a href="#">services</a>
                <a href="#">portfolio</a>
                <a href="#">testimonials</a>
                <a href="#">contact</a>
                <a href="#" id="menu" class="icon">&#9776;</a>
            </div>
        </nav>
    </header>

    <script src="assets/js/script.js"></script>
</body>
</html>