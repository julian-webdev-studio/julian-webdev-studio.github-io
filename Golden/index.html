<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<meta name="keywords" content="Golden, travel, services, work, help, ecomerce, responsiveweb, web_security">
	<title>Golden</title>
	<link rel="stylesheet" href="assets/css/style.css">
	<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700;900&display=swap" rel="stylesheet">
</head>
<body>
	<header  class="header">
		<div class="header__logo">
			<h1><img src="assets/img/logo.png.png" alt="logo"></h1>
		</div>                        
		<!-- Дальше тут будет меню-->
<div class="wrapper">
		<div class="main__welcome">
			<h2>Welcome To Our Studio!</h2>
			<p>It’s nice to Meet you</p>
			<div class="tell__more"><a href="#">Tell Me More</a></div>
		</div>
	</div>	
	</header>
		

	<main>
		<div class="main__services" id="services__link">
			<div class="wrapper">
				<h2>Services</h2>
				<p class="services__p">Proin iaculis purus consequat sem cure. </p>
				<div class="services__cards">
					<div class="services__card">
						<img src="assets/img/icon1.png.png" alt="ecomerce">
						<h2>E-Commerce</h2>
						<p>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt.</p>
					</div>
					<div class="services__card">
						<img src="assets/img/icon2.png.png" alt="resonsive web">
						<h2>Responsive Web</h2>
						<p>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt.</p>
					</div>
					<div class="services__card">
						<img src="assets/img/icon3.png.png" alt="web security">
						<h2>Web Security</h2>
						<p>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt.</p>
					</div>
				</div>
			</div>
		</div>

		<div class="main__portfolio">
			<div class="wrapper">
				<h2>Our portfolio</h2>
				<p>Proin iaculis purus consequat sem cure.</p>
				<div class="galery">
					<a href="#"><img src="assets/img/img1.jpg.png" alt="portfolio"></a>
					<a href="#"><img src="assets/img/img2.jpg.png" alt="portfolio"></a>
					<a href="#"><img src="assets/img/img3.jpg.png" alt="portfolio"></a>
				</div>
			</div>
		</div>
		<div class="main__about">
			<div class="wrapper">
				<h2>ABOUT US</h2>
				<p class="about__p">Proin iaculis purus consequat sem cure.</p>
				<div class="about__cards">
					<div class="about__card">
						
							<div class="about__info">
								<h4>July 2010
								Our Humble Beginnings
								</h4>
								<div class="about__p">
								Proin iaculis purus consequat sem cure 
								digni ssim. Donec porttitora entum suscipit 
								aenean rhoncus posuere odio in tincidunt. Proin 
								iaculis purus consequat sem cure digni 
								ssim. Donec porttitora entum suscipit.
								</div>

							</div>
					<div class="about__photo">
						<img src="assets/img/about1.jpg.png" alt="about" class="mv-100">
					</div>	
					</div> 
				</div>
			</div>
		</div>

	</main>


<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
<script src="assets/js/script.js"></script>
</body>
</html>